client-src/elements/chromedash-form-field.ts

```diff
@@ -1,49 +1,57 @@
-import {LitElement, html, nothing} from 'lit';
+import {SlDetails, SlIconButton, SlInput} from '@shoelace-style/shoelace';
+import {LitElement, TemplateResult, html, nothing} from 'lit';
+import {customElement, property, state} from 'lit/decorators.js';
 import {ref} from 'lit/directives/ref.js';
+
+import {ChromedashApp} from './chromedash-app';
 import './chromedash-textarea';
 import {ALL_FIELDS, resolveFieldForFeature} from './form-field-specs';
 import {getFieldValueFromFeature, showToastMessage} from './utils.js';
+import {Feature, StageDict} from '../js-src/cs-client';
+import {FormattedFeature} from './form-definition';
+
+interface FormFieldValue {
```

Comment:
@jyasskin Could you check if the type is defined correctly here? it seems like the latest change wont pass playwright test